config/logstash.yml

```diff
@@ -64,7 +64,23 @@
 # if there are multiple workers.
 # "false" will disable any extra processing necessary for preserving ordering.
 #
-pipeline.ordered: auto
+# pipeline.ordered: auto
```

Comment:
the default value is `auto`. commenting this out makes it match the rest of the config file where we don't declare default values in the config file we ship.